src/components/FollowerDialog.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <Dialog
+    :open="isOpenFollowersDialog"
```

Comment:
Just curious, why not use `v-if` on the whole component, but a `isOpenFollowersDialog` is used on Dialog open?

Reply:
Use `open` instead of `v-if` to provide a transition on opening